adapters/coap/src/main/java/org/eclipse/hono/adapter/coap/option/TimeOption.java

```diff
@@ -46,21 +46,28 @@ public final class TimeOption extends Option {
      *   <li> <a href="https://github.com/eclipse-hono/hono/pull/3503">Pull request for adding a time option to Hono</a> </li>
      * </ul>
      */
-    public static final int COAP_OPTION_TIME_NUMBER = 0xfde8;
+    public static final int NUMBER = 0xfde8;
 
     /**
      * The request parameter name clients should use to request the server <em>time</em> be sent back as part of the
      * response (as a time option).
      */
-    public static final String COAP_OPTION_TIME_REQUEST_QUERY_PARAMETER_NAME = "hono-time";
+    public static final String QUERY_PARAMETER_NAME = "hono-time";
+
+    /**
+     * Create time option with current system time.
+     */
+    public TimeOption() {
```

Comment:
you now might want to use this very helpful constructor in the CoapContext class ;-)

Reply:
done :)